crates/network-scanner/src/lib.rs

```diff
@@ -23,6 +24,9 @@ pub enum ScannerError {
     #[error("Network interface does not have a netmask")]
     InterfaceDoesNotHaveNetmask(Addr),
 
+    #[error("Network interface does not have a broadcast address")]
+    MdnsError(#[from] mdns_sd::Error),
```

Comment:
question: Is the error message accurate? I don’t know mdns that much, but couldn’t the library return an error for many other reasons unrelated to the broadcast address?

Reply:
updated